lib/utils/token/token.go

```diff
@@ -0,0 +1,80 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package token
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/lib/defaults"
+)
+
+// Verify ensures the token fits into our length requirements and
+// its bits of entropy are sufficient.
+func Verify(token []byte) error {
+	return verify(token, defaults.MaxTokenLength)
+}
+
+// VerifyHashed ensures the token fits into our length requirements and
+// its bits of entropy are sufficient. If the token is not going to be
+// hashed by bcrypt before it will be used, use [Verify] instead.
+func VerifyHashed(token []byte) error {
+	return verify(token, defaults.MaxHashedTokenLength)
+}
+
+func verify(token []byte, maxLen int) error {
+	if len(token) < defaults.MinTokenLength {
+		return trace.BadParameter("token is too short, min length is %d", defaults.MinTokenLength)
+	}
+	if len(token) > maxLen {
+		return trace.BadParameter("token is too long, max length is %d", maxLen)
+	}
+
+	entropyBits := TokenStrength(token)
+	fmt.Println(entropyBits)
```

Comment:
```suggestion
```

Reply:
Thanks, removed it and then added it back again to test something